docs/process-model.md

```diff
@@ -0,0 +1,116 @@
+description, information if it is private and at least one moderator. It can have an image and, a
+background image. If it is private it also might have participants.
+
+In euth a project is made up by a collection of particiaption modules. Each participation module has
+phases that can be activated or deactivated. For our example there exists only one participation
+module called 'idea collection'. It offers the following phases:
+
+  1. Add ideas:
+   - particpants can add ideas
+  2. Add and comment ideas:
+   - participants can add ideas
+   - participatns can comment ideas
+  3. Offline idea workshop:
+   - moderator can upload ideas from offline meeting
+  4. Comment ideas:
+   - participatns can comment ideas
+  5. Rate and commment ideas:
+   - participatns can comment ideas
+   - participatns can vote ideas
+  6. Rate ideas:
+   - participants can rate ideas
+  7. Offline rate:
+   - moderator can upload rates from offline meeting
+  8. Offline jury selection:
+   - moderator can upload labels/statments from offline jury meeting
+
+When a participation module is added to a project all phases will be added. The intiator can then disable
+phases that she doesn't need. She can also set a title, set a desription, add start date, and
+add end date for each phase.
+
+Start and end dates of a phase should never overlap with another phase of all participation modules
+in the same project. The might be ommited, in that case the phase needs to be manually switches. If that
+happens the end date of the prvious phase and the start date of the new phase will be set. The active
+phase is that phase that has a start date in past and a end date in the future or a start date in the past 
+but no end date set. There can be one or no active phase.
+
+To make this less theoretical, lets give the representation of an example project:
+
+- project
+   - Name: Build a new tree house.
+   - Description:
+      We want a new tree house in our garden and invite all neighbours and family members to design that tree house.
+      
+      In order to be cheap and sustainable the tree house should be build from recycleable
+      materials.
+      
+      First we collect cool ideas for elements that a tree house should have (like rope ladder, fountain, 
+      bucket shower).
+       
+      Than you can add your proposal by drawing a concept study. Through rate and jury
+      decision the best one will be selected and than realized.
+   - Moderators: Tina, Sam
+   - Participation modules:
+       - Idea Collection:
+         - Phases:
+             1. Add ideas:
+               -Title: Best tree house element
+               - Description:
+                  Give us your facourite tree house element and say why it is the best.
+               - Start: 1st june 2016
+               - End: 5th june 2016
+             2. Add and comment ideas: **Disabled**
+             3. Offline idea workshop: **Disabled**
```

Comment:
It would be not possible to add other peoples ideas as well online after the f2f workshop took place, is that correct? I wonder if that is a use case.....